nodejs/eks/cni.ts

```diff
@@ -131,6 +151,16 @@ function computeVpcCniYaml(cniYamlText: string, args: VpcCniInputs): string {
     } else {
         env.push({name: "AWS_VPC_K8S_CNI_LOG_FILE", value: "stdout"});
     }
+    if (args.vethPrefix) {
+        env.push({name: "AWS_VPC_K8S_CNI_VETHPREFIX", value: args.vethPrefix.toString()});
+    } else {
+        env.push({name: "AWS_VPC_K8S_CNI_VETHPREFIX", value: "eni"});
```

Comment:
Do you actually need to set this? It should already default to this value, right?

Reply:
> It should already default to this value

Right. Though, historically, we've set defaults for envvars that are configurable in pulumi/eks to aid the user in understanding top-level args, and bubbling them up in docs.

Reply:
Going to continue with this as its the same pattern we currently use. Will revisit envvars in https://github.com/pulumi/pulumi-eks/issues/337